config/environments/production.rb

```diff
@@ -117,4 +117,18 @@
   # config.active_record.database_selector = { delay: 2.seconds }
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
+
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  host = '<your heroku app>.herokuapp.com'
```

Comment:
using environment variable to save host value
it's the same below:
```
 user_name: ENV['SENDGRID_USERNAME'],
 password: ENV['SENDGRID_PASSWORD']
```

Reply:
![Screenshot from 2021-09-06 17-32-48](https://user-images.githubusercontent.com/87748931/132195231-fc62057c-3f61-487a-83a8-e09b08675bd1.png)
is it correct?